### DISCUSSION TITLE
Managing the Discussions!

### DISCUSSION THREAD

**Discussion by Coffee2Bits** [DevOps-MBSE/AaC#78] [Archive] [answered]
Apparently, Github doesn't currently support cleaning up discussions other than deleting them. See [Github issue #3097](https://github.com/github/feedback/discussions/3097).

Looking at our current discussions, I like how the Docstring Convention #49 discussion at least has a green checkmark giving a visual indication that the discussion is settled/question answered. I looked at some options for how to manage the old discussions without deleting them and I have two ideas:
 
1. Create an 'Archived' category for discussions.

2. Edit the existing categories (ideas and general) to implement the `Question / Answer` discussion format to make use of the "answered" function and accompanying green checkmark.

**Comment by jondavid-black** [comment]
I like this idea.  I like how we've been able to collaborate using discussions, but I wish there was a clear visual when a discussion had content I haven't read yet.  I went ahead and created an "Archive" category to test out idea 1 above.  Let's see if it gives us what we want.

**Comment by jondavid-black** [comment, marked as answer]
I created the Archive category and put a discussion into it.  It really didn't clean anything up.

The next thing I tried was a label called "resolved".  With the label you can filter out resolved discussions pretty easily by Alt+'click the label in the filter drop down'.  That worked for me.

What do you guys think about just labeling our resolved discussions?

**Reply by cvchaparro** [reply]
I like the label method so that, at least, we can look at discussions that are still relevant. Additionally, once we can finally close discussions, we could probably write a script that keys off of that label to go through and close all of the discussions that are "resolved".

**Reply by Coffee2Bits** [reply]
I agree, I like the label for filtering. Eventually, the resolved items will sink to the bottom because of a lack of engagement, but being able to just filter them out altogether helps declutter it a bit. 

I just wish there was a way other than bookmarks to apply that filter by default, but I haven't found one yet.

**Reply by Coffee2Bits** [reply]
That being said, I do also like the ability to mark an answer with the archive category. It makes it pretty clear what the solution/chosen path is especially in long threads.

**Comment by lizzcondrey** [comment]
Woo, GitHub has addressed this issue and now provides a way of closing discussions. 

So, I am commenting to those that are marked resolved with confirmation things are in place and closing. I am commenting on those that I think are eligible for closure with request for confirmation. And I am commenting on those remaining with current status to bring attention back around. I will be making my way through all current discussions this sprint.
